src/renderers/dom/client/eventPlugins/__tests__/EnterLeaveEventPlugin-test.js

```diff
@@ -37,11 +37,11 @@ describe('EnterLeaveEventPlugin', function() {
     var iframeDocument = iframe.contentDocument;
 
     iframeDocument.write(
-      '<!DOCTYPE html><html><head></head><body></body></html>'
+      '<!DOCTYPE html><html><head></head><body><div></div></body></html>'
```

Comment:
This change is for the `Rendering components directly into document.body` warning. 

Without doing this, I need to spyOn the `console.log`. or should I ?